docs/radicle.md

```diff
@@ -28,39 +25,50 @@ while offering global decentralized repository storage through the peer-to-peer
 networking layer.
 
 Since all data on the network is stored locally by peers on the network,
-develops can share and collaborate on Git repositories *without* relying on
+developers can share and collaborate on Git repositories *without* relying on
 intermediaries such as hosted servers.
 
-For more on Radicle Link, check out the [Understanding Radicle][wh] section.
+The radicle-link protocol distinguishes between two types of identities: personal and project.
+The first describes an actor in the system, while the second describes a (software) project
+on which one or more actors collaborate. In Radicle:
 
-## How is Radicle different from GitHub?
+1. peers follow other peers.
+2. peers track projects they are interested in.
+3. peers gossip about projects. This means tracking and replicating objects from
+the peers they follow, about the projects they are interested in.
 
-When coming to Radicle from a centralized code collaboration platform like
-Github or Gitlab, you might notice that Radicle priotizes different concerns. In
-contrast to the traditional Web2.0 model of centralized code collaboration
-platforms (i.e. global by default, reliance on service providers, convienience),
-Radicle ensures that:
-
-1. People control their identity
-2. People control their content
-3. People control their social interactions
+It's this "trusted" social graph of peers and projects that becomes the foundation for
+collaboration within Radicle.
 
-These concerns have informed the design of [Radicle Link][rl], the peer-to-peer
-protocol that powers the Radicle network. Radicle Link gossips and replicates
-data between `peers`.
+For more on Radicle Link, check out the [Understanding Radicle][wh] section.
 
-In Radicle:
+## How is Radicle different from GitHub?
 
-1. Peers follow other peers.
-2. Peers track projects they are interested in.
-3. Peers gossip about projects - this means tracking peers can replicate objects
-   from the peers they follow, about the projects they are interested in.
+When coming to Radicle from a centralized code collaboration platform like
+Github or Gitlab, you might notice that Radicle priotizes different concerns. Concretely:
```

Comment:
This language makes it seem that Radicle is replacing GitHub as you are "coming to it". Let's rephrase the language. Also the "being in" phrase in the bullet point section reads weird.

```suggestion
When comparing Radicle to centralized code collaboration platforms like GitHub or Gitlab, it's important to note that Radicle prioritizes different concerns. Concretely:

- People should control their identity
- People should control their content
- People should control their social interactions

```